src/raspbian/12/helix/arm32v6/Dockerfile

```diff
@@ -60,7 +62,7 @@ ENV LANG=en_US.utf8
 RUN ln -sf /usr/bin/python3 /usr/bin/python && \
     pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
     export CRYPTOGRAPHY_DONT_BUILD_RUST=1 && \
-    pip install ./helix_scripts-*-py3-none-any.whl
+    pip install ./helix_scripts-*-py3-none-any.whl --break-system-packages
```

Comment:
please add a comment why we need --break-system-packages

Reply:
Alternatively, could we use the multistage docker + python venv approach we've done in all the other images and avoid using `--break-system-packages` (#1142)?